src/hotspot/share/utilities/resizeableResourceHash.hpp

```diff
@@ -30,7 +30,7 @@
 template<
     typename K, typename V,
     AnyObj::allocation_type ALLOC_TYPE,
-    MEMFLAGS MEM_TYPE>
+    MemTag MEM_TYPE>
```

Comment:
I think s/MEM_TYPE/mem_type/, but other non-type template parameters here are also
all-uppercase, so I guess better to leave it for now and look at it later.

Reply:
Yes, I followed the local pattern.